README.md

```diff
@@ -151,29 +157,39 @@ separately.
 
 ## Provisioning TheRock 🪨
 
-In order to provision TheRock using either a developer/automated nightly release or a specific CI runner build, use the `build_tool/provision.py` script.
+In order to provision TheRock using either a developer/automated nightly release, a specific CI runner build or already existing installation of TheRock, use the `build_tool/provision_machine.py` script.
 
 Provisioning script setup:
 
 - `python3 -m venv venv`
 - `source venv/bin/activate`
 - `pip install -r requirements.txt`
-- `python build_tools/provision.py --help`
+- `python build_tools/provision_machine.py --help`
 
 Examples:
 
-- `python build_tools/provision.py --run-id 14474448215 --amdgpu-family gfx94X-dcgpu`: Downloads the gfx94X S3 artifacts from GitHub CI workflow run 14474448215 to the default output directory `therock-build`
+- Downloads the gfx94X S3 artifacts from GitHub CI workflow run 14474448215 (from https://github.com/ROCm/TheRock/actions/runs/14474448215) to the default output directory `therock-build`:
 
-<br/>
+```
+python build_tools/provision_machine.py --run-id 14474448215 --amdgpu-family gfx94X-dcgpu
+```
```

Comment:
indentation is load bearing here, to keep code blocks nested under each list item
```suggestion
    ```
    python build_tools/provision_machine.py --run-id 14474448215 --amdgpu-family gfx94X-dcgpu
    ```
```